docker/rootfs/usr/local/go2rtc/create_config.py

```diff
@@ -0,0 +1,35 @@
+"""Creates a go2rtc config file."""
+
+import json
+import os
+import yaml
+
+
+FRIGATE_ENV_VARS = {k: v for k, v in os.environ.items() if k.startswith("FRIGATE_")}
+config_file = os.environ.get("CONFIG_FILE", "/config/config.yml")
+
+# Check if we can use .yaml instead of .yml
+config_file_yaml = config_file.replace(".yml", ".yaml")
+if os.path.isfile(config_file_yaml):
+    config_file = config_file_yaml
+
+with open(config_file) as f:
+    raw_config = f.read()
+
+if config_file.endswith((".yaml", ".yml")):
+    config = yaml.safe_load(raw_config)
+elif config_file.endswith(".json"):
+    config = json.loads(raw_config)
+
+go2rtc_config: dict[str, any] = config["go2rtc"]
+
+if not go2rtc_config.get("log", {}).get("format"):
+    go2rtc_config["log"] = {"format": "text"}
+
+if not go2rtc_config.get("webrtc", {}).get("candidates", []):
+    go2rtc_config["webrtc"] = {"candidates": ["stun:8555"]}
+
+for name in go2rtc_config.get("streams", {}):
+    go2rtc_config["streams"][name] = go2rtc_config["streams"][name].format(**FRIGATE_ENV_VARS)
```

Comment:
go2rtc already supports environment variable substitution: https://github.com/AlexxIT/go2rtc/wiki/Configuration

I think this may conflict.

Reply:
Okay, can just tear that logic out then since go2rtc will handle it anyway